### DISCUSSION TITLE
[pun] ChanL's Possible Futures

### DISCUSSION THREAD

**Discussion by adlai** [adlai/chanl#7] [General]
### Is your idea related to a problem?
Please describe how you reached this [line](https://github.com/fstamour/breeze/blame/main/breeze.asd#L23) and why your problem is best solved by ChanL rather than a different/separate library.

### Describe the solution you'd like
Please describe the improvement or new feature [you'd](https://github.com/GunioRobot/Eager-Future2/blob/master/scheduler.lisp#L38-L46) like added to ChanL.

### Describe alternatives you've considered
If relevant, describe how you currently [deal with the problem](../../../favicon.ico), and/or how you'd proceed if ChanL does not accept your suggestion.

### Additional context
If relevant, include additional information.

**Comment by adlai** [comment]
tl;dr - there are a bunch of parallelism libraries already, so it's worth surveying this project's dependents before delivering major new chunks of code.
